projecttacoma/fqm-testify
# App skeleton and Mantine setup

# Summary
Creates basic NextJS React app with TypeScript support. Adds basic [Mantine](https://mantine.dev/) setup to the project with a header bar and light/dark mode switch.

## New Behavior
The user can now run and navigate to the application. The application on port 3000 in dev mode.

## Code Changes
* Adds basic app skeleton using `create-next-app`
* Adds various `mantine` dependencies as part of the initial Mantine setup
* Adds Jest dependencies

# Testing Guidance
* Start the app in dev mode with `npm run dev`
* Navigate to `http://localhost:3000` to see the skeleton app
* Run unit test with `npm run test`

Compare this set-up to the current files and file structure of the [bulk-submit-demo](https://github.com/projecttacoma/bulk-submit-demo) repository.
